.github/workflows/software-versions.yaml

```
@@ -62,6 +56,12 @@ jobs:
           mkdir -p "$HOME/.docker"
           cp ci/secrets/pull-secret.json "$HOME/.docker/config.json"
 
-      - name: Validate manifest annotations against image SBOMs
+      - name: Configure Quay.io API auth for RHOAI namespace
+        if: secrets.RHOAI_QUAY_BOT_USERNAME != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)